calculator-app/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from "react"
+import Screen from "./Components/Screen"
+import Buttons from "./Components/Buttons"
+import numeral from "numeral"
+import "./App.css"
+
+export class App extends Component {
+  constructor(){
+    super()
+
+    this.state = {
+      displayValue: "0",
+      prevValue: "",
+      operation: ""
+    }
+  }
+  
+  handleDisplay = (e) => {
+    this.setState({ displayValue: numeral((this.state.displayValue.concat(e.target.value))).format('0,0') })
+  }
+  
+  handleNegative = () =>{
+    this.setState({ displayValue: numeral(this.state.displayValue.split(',').join("") * -1).format('0,0') })
+  }
+  
+  handleOperation = (e) => {
+    this.setState( prevState => ({ prevValue: prevState.displayValue, displayValue: "", operation: e.target.value}))
+  }
+
+  handleResult = () => {
+    if(this.state.operation === "+"){
+      this.setState({ displayValue: String(Number(this.state.prevValue.split(',').join("")) + Number(this.state.displayValue.split(',').join(""))) })
+    } else if(this.state.operation === "-"){
+      this.setState({ displayValue: String(this.state.prevValue.split(',').join("") - this.state.displayValue.split(',').join("")) })
```

Comment:
The duplicate sections on this line and line 36 can be dried out by storing in a variable.